zebrad/src/components/sync/end_of_support.rs

```diff
@@ -22,8 +22,8 @@ pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_626_500;
 ///
 /// - Zebra will exit with a panic if the current tip height is bigger than the `ESTIMATED_RELEASE_HEIGHT`
 ///   plus this number of days.
-/// - Currently set to 14 weeks.
-pub const EOS_PANIC_AFTER: u32 = 70;
+/// - Currently set to 5 weeks to end support before Mainnet Nu6 activation at block [`2_726_400`](https://zips.z.cash/zip-0253).
+pub const EOS_PANIC_AFTER: u32 = 35;
```

Comment:
Nu6 activation height is in about 41 days from now:

2726400 - 2678363 = 48037 % 1152 = 41,6

I changed the support of this release to 5 weeks to end about 1 week before the activation height. I am open to other alternatives.